spec/controllers/search_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe SearchController, type: :controller do
+
+  describe '#search' do
+
+      let (:tech2) { FactoryGirl.create(:technology, name: 'tech2')}
+      let (:techs) { [FactoryGirl.create(:technology, name: 'tech1'), tech2]}
+      let (:techs2) { [FactoryGirl.create(:technology, name: 'tech3'), tech2] }      
+      let! (:organization_1) {FactoryGirl.create(:published_organization, name: "test organization", tech_team_size: 10, employee_count: 20, technologies: techs)}
+      let! (:organization_2) {FactoryGirl.create(:published_organization, name: "yyyo", tech_team_size: 40, employee_count: 50, technologies: techs2)}
```

Comment:
don't put a space after `let` and `let!`. Also to be consistent, put a space after the `{` in the blocks.